setup.py

```diff
@@ -116,7 +116,8 @@ def girder_install(self, component):
             'girder-version.json',
             'conf/girder.dist.cfg',
             'mail_templates/*.mako',
-            'mail_templates/**/*.mako'
```

Comment:
I suppose you could say this about a lot of places though. For right now, it's pointless cruft.

If anyone does want to add subdirectories in the future, they can just update the install script again accordingly; both these changes would have to be part of a commit to Girder proper, so this doesn't impact plugins or any development API.